test/helm_task_test.go

```diff
@@ -42,13 +45,17 @@ var (
 // TestHelmDeployPipelineRun is an integration test that will verify a pipeline build an image
 // and then using helm to deploy it
 func TestHelmDeployPipelineRun(t *testing.T) {
+	if testEnv.Platform == "linux/s390x" {
+		t.Skipf("skip for s390x architecture")
+	}
```

Comment:
May be it's better to have 1 entry point for different e2e tests to skip, which can be extended for other architectures? For instance in `setupWithCleanup`?

Reply:
So I want to make the `skip` very explicit *per* tests. Aka, if a test needs to be skip for whatever reason, it needs to be known when reading the test. In `docker/docker` we used the concept of `requirements` (`requirements.IsNetworkAvailable`, …)

Reply:
ok, I have to say I like to keep related things in one place, just not to spam the code with `if` statements for different cases (and many archs in the future), however I'd like to believe that all tests will work for all archs 🤞 and there won't be many exclusions required..

Reply:
example in `docker/docker`: https://github.com/moby/moby/blob/master/integration/container/create_test.go#L74